_posts/2017-09-29-make-connections.html

```diff
@@ -0,0 +1,46 @@
+<h3>"Alexa, tell me about &lt;CompanyName&gt;"</h3>
+<p>We have explored the possibilities to develop for two major players: Amazon and Google. At the time of trying out the Echo Dot, development was still limited and a very manual task of filling in data. We were also limited by our own source of data for companies, not having it readily available for consumption. We decided to switch to Google, which was slightly more flexible with dynamic data.</p>
+
+<h2>"OK Google"</h2>
+<p>FT Labs has a longer term goal where we want to allow rich conversations to take place between our audience and voice devices, this game is a step along the path to that capability.</p>
+<p>To develop Make Connections, we used <a href="https://api.ai/">API.ai</a> and its Action on Google integration. The game uses the correlations service, based on our previous exploration of the <a href="https://labs.ft.com/2015/07/six-degrees-of-angela-merkel/">6 degrees of Angela Merkel</a>.</p>
+<p>In the same spirit, players much choose from 3 propositions which person has been mentioned in the same FT article as the one in the question. The game only looks at recent articles (published within a week), so the answers will be different over time.</p>
+<blockquote>
+	<p>"Theresa May was mentioned in an article with which one of the following people?"<p>
+	<ol>
+		<li>Martin Gruenberg</li>
+		<li>Richard Cousins</li>
+		<li>Mark Carney</li>
+	</ol>
+	<p><cite>Correct answer on 02/10/2017: Mark Carney</cite></p>
+</blockquote>
+<p>An answer can be made by saying the option number or the person's name. Upon answering, the player is offered more context on the connection: the title of the article where the correlation occured. If the answer was correct, another question comes up and the score increases until a wrong answer is given, or no more connections are found.</p>
+<p>The game has also been developed to work on Google Assistant on mobile devices. On this surface, users will be able to click on a name to give their answer, and access a link to the relevant FT article.</p>
+<p>API.ai is a powerful interface where one can easily create intents, upload entities (possible user inputs) and have a basic Action ready within minutes. From a developer's point of view, though, it is a strange mix of <abbr title="Graphical User Interface">GUI</abbr> and externally hosted service. It would be preferrable to be able to generate everything in the code in that case, and have an endpoint to tap into the Machine Learning side of the Assistant.</p>
+
+<h2>Limitations</h2>
```

Comment:
IMHO, we have two types of limitations:
1) for the game
- inability to pre-load with our large corpus of known names
- inability do our own fuzzy matching on what the user might have said
- some people's voices just not being parsed well, even on simple terms such as one two three
- the need to add numbers to the list of options
- ...
2) for VUIs in general
- interruptions
- the mix of GUI+external service
- ...